### PR TITLE
Update edit_type.yaml

### PR DESCRIPTION
**Pull request by dartar**
Italian translation messages for edit type campaign

### CODE REVIEW

File: forms/available/edit_type.yaml [hunk 1]
```diff
@@ -164,69 +164,77 @@ i18n:
...
-    semantic-labels-label: Scegli un significato semantico
+    semanticlabels-label: Scegli l'intenzione più probabile per questa modifica.
     # Semantic meanings
-    link-disambig-label: Collegamento disambiguazione
-    link-disambig-description:
-    fact-update-label: Fatto Aggiornamento
-    fact-update-description:
-    formatting-label: Formattazione
-    formatting-description:
+    copyediting-label: Copy Editing
```
Comment (halfak) [inline forms/available/edit_type.yaml:169]: Still English.  Is that OK?

**Comment by halfak** [comment]
Other than the one note -- which I'm guessing is actually OK since it was that way in the old form.  Once @dartar confirms, this is ready to merge.

File: forms/available/edit_type.yaml (hunk 1 above)
Comment (dartar) [inline forms/available/edit_type.yaml:169]: It's commonly used in Italian but let me get a second opinion before closing this.

**Comment by dartar** [comment]
Quick update: I ran this past a group of IT Wikipedians and there's disagreement on the best label to choose. I'll let you know when consensus is reached.

**Pull request merged**